### PR TITLE
Added some updates on the prop name (height, color), export and readme file

### PR DESCRIPTION
**Pull request by azouaoui-med**
First, i want to say thank you for sharing this awesome library, i love your work, and that motivated me to contribute even with a small thing

This PR contains updates which i thought will improve the usability :

- Changed the name of the prop height to size to avoid confusion since it is used for both height and width

- Changed the name of the prop color of the item props to backgroundColor since it is used as a background color and to keep the consistency of naming

- added extra exports for the main component to make it easy to use when imported

  so instead of using this type of import

```
  import FloatingButton from "react-floating-button";
  import Item from "react-floating-button";
```

you can also use

```
import { FloatingButton, Item } from "./FloatingButton";
```

- added .prettierrc so the code formatting don't change, especially when others contribute to your code, this will make the PRs readable

- updated the readme file, i added table for props definition and simplified the example for usage

:)

### CODE REVIEW

File: FloatingButton/index.js [hunk 1]
```diff
@@ -144,10 +144,12 @@ FloatingButton.defaultProps = {
...
-  height: PropTypes.number,
+  size: PropTypes.number,
   top: PropTypes.bool,
   right: PropTypes.bool,
   children: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
 };
 
 export default FloatingButton;
+export { FloatingButton };
+export { FloatingButton as Item };
```
Comment (na6im) [inline FloatingButton/index.js]: The Item is not The Floating button it’s exported from styles.js or Item.js

File: README.md [hunk 2]
```diff
@@ -32,52 +32,65 @@ Live demo: https://floating-button.namokrane.works/
...
-$ yarn add react-floating-button
+yarn add react-floating-button
 ```
 
 #### Usage
 
 ```javascript
-import FloatingButton from "react-floating-button";
-import Item from "react-floating-button";
+import { FloatingButton, Item } from "./FloatingButton";
```
Comment (na6im) [inline README.md]: From ‘react-floating-button’

**Comment by na6im** [comment]
Thank you for your contribution I appreciate it, but there is some small changes to do just check my comments

**Comment by azouaoui-med** [comment]
If you have added comments then they are not visible to me at the moment

**Review by na6im — CHANGES_REQUESTED** [review]

**Comment by na6im** [comment]
> If you have added comments then they are not visible to me at the moment

Sorry my bad, now they are visible

File: FloatingButton/index.js (hunk 1 above)
Comment (azouaoui-med) [reply FloatingButton/index.js]: Ah i see, i thought it was form the FloatingButton when i saw this `import Item from "react-floating-button"`, but its fixed now

File: README.md (hunk 2 above)
Comment (azouaoui-med) [reply README.md]: This what happens when you copy past lol , fixed

Comment (na6im) [reply README.md]: Thank you Mohammed

**Review by na6im — APPROVED** [review]
Thank you

**Pull request merged**